localization_flow/jtlocalize/genstrings.py

```diff
@@ -99,11 +92,10 @@ def generate_strings(project_base_dir, localization_bundle_path, tmp_directory,
 
     genstrings_rc = genstrings_process.returncode
     if genstrings_rc != 0:
-        logging.fatal("genstrings returned %d, aborting run!", genstrings_rc)
+        logging.fatal("\n!!!!!!!!! genstrings returned %d, ABORTING RUN !!!!!!!!!\n", genstrings_rc)
```

Comment:
I wonder if you can make it red :)

Reply:
Could be nice, but I'll leave it for next time